Projects/Embark/Sources/EmbarkState.swift

```diff
@@ -34,6 +34,10 @@ public struct EmbarkState {
     var passageNameSignal: ReadSignal<String?> {
         currentPassageSignal.map { $0?.name }
     }
+    
+    var passageTooltipSignal: ReadSignal<Bool> {
```

Comment:
How about making this return the tooltip instead? Then you could just `if tooltip == nil { hideTabBarButton }`